### PR TITLE
translate run_scripts.md to chinese

### PR DESCRIPTION
**Pull request by jiaqiw09**
# What does this PR do?

Part of  #26803 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? _not necessary_

## Who can review?
@stevhliu

### CODE REVIEW

**Comment by jiaqiw09** [comment]
@stevhliu

hi,

here is another PR of run_scripts.md, and I will fix merge conflict later.

Best

File: docs/source/zh/run_scripts.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+```bash
+pip install -r requirements.txt
+```
+
+## 运行脚本
+
+<frameworkcontent>
+<pt>
+
+示例脚本从huggingface的[Datasets](https://huggingface.co/docs/datasets/)库下载并预处理数据集。然后，脚本通过[Trainer](https://huggingface.co/docs/transformers/main_classes/trainer)使用支持摘要任务的架构对数据集进行微调。以下示例展示了如何在[CNN/DailyMail](https://huggingface.co/datasets/cnn_dailymail)数据集上微调[T5-small](https://huggingface.co/t5-small)。由于T5模型的训练方式，它需要一个额外的`source_prefix`参数。这个提示让T5知道这是一个摘要任务。
```
Comment (stevhliu) [inline docs/source/zh/run_scripts.md:91]: Should the translation be like this to match the TensorFlow one below?

```suggestion
示例脚本从 🤗 [Datasets](https://huggingface.co/docs/datasets/)库下载并预处理数据集。然后，脚本通过[Trainer](https://huggingface.co/docs/transformers/main_classes/trainer)使用支持摘要任务的架构对数据集进行微调。以下示例展示了如何在[CNN/DailyMail](https://huggingface.co/datasets/cnn_dailymail)数据集上微调[T5-small](https://huggingface.co/t5-small)。由于T5模型的训练方式，它需要一个额外的`source_prefix`参数。这个提示让T5知道这是一个摘要任务。
```

**Review by stevhliu — APPROVED** [review]
Very nice! 👍

**Comment by jiaqiw09** [comment]
@stevhliu 

reviews and merge conflict have been sloved.

best

**Pull request merged**